src/WebJobs.Extensions.Http/HttpRequestExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Internal;
+using Microsoft.AspNetCore.Mvc.Formatters;
+
+namespace Microsoft.Azure.WebJobs.Extensions
```

Comment:
Was the change in namespace intentional on these http extensions? Used to be under .Http before